.cloudbuild/ci/integration-tests.yaml

```diff
@@ -0,0 +1,29 @@
+steps:
+  # Run non-root integration tests. The root user is selected by docker image.
+  - name: us-docker.pkg.dev/ci-account/teleport/buildbox-root:v0.1.0
```

Comment:
What is `us-docker.pkg.dev`?

Reply:
Google cloud build artifact registry. Buildbox images get built on GCB and injected directly into that.

Reply:
We also have buildbox images here https://quay.io/repository/gravitational/teleport-buildbox?tab=tags, I don't know if there is a preferred registry to use.

Reply:
I put the buildboxes in the artifact registry for a couple of reasons:

1. To be close to the build serves and cut down the pull time on every build
2. To avoid having to deal with extra credentials when pushing the images from the build-site to the registry-site.

Reply:
How and where are these images generated?

Reply:
Generated from the `quay.io` images based on dockerfiles [here](https://github.com/gravitational/teleport-gcb-buildbox). 

The actual image construction is done by GCB using the `Buildbox-(Non)?Root-Update` triggers, and the resulting image is pushed to `us-docker.pkg.dev/ci-account/teleport/buildbox-(non)?root` on a successful build.